mcs/class/System.Private.CoreLib/System.Collections.Generic/EqualityComparer.cs

```diff
@@ -1,17 +1,58 @@
+using System.Runtime.CompilerServices;
```

Comment:
EqualityComparer.cs isn't added to `System.Private.CoreLib.csproj` - so this is dead code right now.

Reply:
It is there:
      <Compile Include="System.Collections.Generic\EqualityComparer.cs" />